14/umbraco-cms/reference/management-api/README.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Get started with the Management API.
+---
+
+# Management API
+
+The Management API delivers headless capabilities built directly into Umbraco. Unlike the Content Delivery API, the Management API enables interaction with everything else in the backoffice except for the Content and Media items.
+
+The API returns responses in a JSON format, which can be presented in different channels using your preferred technology stack. This feature maintains the user-friendly editing experience of Umbraco, while also ensuring an efficient delivery of content in a headless manner. Additionally, with its different extension points, the API can be customized to meet a wide range of requirements.
+
+The Management API can also be used for Custom apps or Workflows with OpenID Connect.
+
+{% hint style="info" %}
+The Management API is a replacement for the backoffice controllers that lacked RESTful capabilities.
+{% endhint %}
+
+## Swagger Documentation
+
+Umbraco ships with Swagger to document the Management API. Swagger and the Swagger UI are based on [Swashbuckle.AspNetCore](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/) and is available at `{yourdomain}/umbraco/swagger`. For security reasons, both are disabled in non-production environments.
+Read more about Swagger in the [API versioning and OpenAPI](../api-versioning-and-openapi.md) article.
+
+The Swagger documentation allows you to select a definition and go to either Umbraco Management API or Content Delivery API. You can also create custom documentation in Swagger by referring to the [Custom Swagger API with Authorization](../custom-swagger-api.md) article.
+
+![Umbraco Management API documentation in Swagger](../images/management-api-swagger.png)
+
+In the Swagger Umbraco Management API, you can find a collection of available endpoints in this version of Umbraco. It also includes all the legacy APIs that are available for backward compatibility.
```

Comment:
```suggestion
In the Swagger Umbraco Management API, you can find a collection of available endpoints in this version of Umbraco. 
```

We won't really be including legacy APIs from the get-go --- if anything we should remove the comment in Swagger that states so. Down the line though, legacy (obsolete) APIs will be included, as we evolve the API.